app/app.go

```diff
@@ -89,6 +90,7 @@ var (
 
 	Upgrades = []upgrades.Upgrade{v4.Upgrade, v5.Upgrade, v7.Upgrade, v9.Upgrade}
 	Forks    = []upgrades.Fork{v3.Fork, v6.Fork, v8.Fork, v10.Fork}
```

Comment:
Then, logic in `app/forks.go` can be reverted

```suggestion
	Forks    = []upgrades.Fork{test.Fork, v3.Fork, v6.Fork, v8.Fork, v10.Fork}
```

Reply:
If I did this though, wouldn't it run at block 60 on mainnet without the app/forks.go logic?

Reply:
yeah I did not consider the fact that there are use cases for replaying from 0. I think other discussed suggestions with build tags/genesis tweaks are more solid options than this